.planning/milestones/v1.1-phases/02-post-api-notify-endpoint-with-privacy-hardening/02-01-PLAN.md

```
@@ -0,0 +1,396 @@
+<task type="auto">
+  <name>Task 1: Update FcmPush constructor to accept Arc&lt;reqwest::Client&gt;</name>
+  <files>src/push/fcm.rs</files>
+  <read_first>
+    - src/push/fcm.rs (full file, ~310 lines — see lines 1-14 for imports, 43-83 for struct + new(), 95-158 for get_access_token, 218-305 for send_to_token)
+    - .planning/milestones/v1.1-phases/02-post-api-notify-endpoint-with-privacy-hardening/02-CONTEXT.md (sections "Outbound Client Hygiene" D-07 + D-08, "Code Insights" / "Integration Points")
+    - .planning/milestones/v1.1-phases/02-post-api-notify-endpoint-with-privacy-hardening/02-PATTERNS.md (section "MODIFY src/push/fcm.rs", subsection "Analog 1 — constructor cascade")
+  </read_first>
+  <action>
+    Modify `src/push/fcm.rs` per D-07/D-08:
+
+    1. **Imports (line 6)**: keep `use std::sync::Arc;` as-is (already present). The `use reqwest::Client;` at line 3 stays as-is — `Client` remains the inner type, `Arc<Client>` wraps it.
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)